alfalfa_worker/worker_fmu/add_site.py

```diff
@@ -253,18 +253,18 @@ def add_fmu(self):
         old version of the Modelica Buildings Library and JModelica.
         :return:
         """
-        self.add_site_logger.logger.info("add_fmu for {}".format(self.key))
+        self.logger.info("add_fmu for {}".format(self.key))
 
-        self.ac.s3_bucket.download_file(self.key, self.fmu_path)
+        self.s3_bucket.download_file(self.key, self.fmu_path)
 
         # External call to python2 to create FMU tags
-        call(['python', 'lib/fmu_create_tags.py', self.fmu_path, self.file_name, self.fmu_json])
+        call(['python', 'alfalfa_worker/lib/fmu_create_tags.py', self.fmu_path, self.file_name, self.fmu_json])
```

Comment:
I remove the `os.chdir` at the beginning of the `worker.py` which means that we need to path from the root of the python worker project.

Reply:
I hope that this disappears when we move to python3 and classify all of the classes.